tests/functional/spec/placeholder_option_label_from_value.spec.js

```diff
@@ -0,0 +1,33 @@
+import MandatoryCheckboxPage from "../generated_pages/placeholder_option_label_from_value/mandatory-checkbox.page";
```

Comment:
https://github.com/ONSdigital/eq-questionnaire-runner/pull/756 introduced a `test/functional/spec/features/placeholder/` folder. We should move this to that.